<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>ĐỔI NGOẠI TỆ SANG NỘI TỆ (VND)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>541</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(224, 235, 255);</string>
    </property>
    <property name="title">
     <string>Đổi ngoại tệ sang nội tệ:</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>52</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Ngoại tệ:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxNgoaiTe">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 208, 239);</string>
     </property>
     <item>
      <property name="text">
       <string>USD (Đô la Mỹ)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EUR (Đồng Euro)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GBP (Bảng Anh)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SGD (Đô la Sing)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AUD (Đô la Úc)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CNY (Nhân dân tệ)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>JPY (Yên Nhật)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>KRW (Won)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>THB (Baht Thái)</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Số ngoại tệ muốn đổi: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSoLuongNgoaiTe">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 208, 239);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>VNĐ: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditVND">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 208, 239);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>541</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 251, 225)</string>
    </property>
    <property name="title">
     <string>Các chức năng:</string>
    </property>
    <widget class="QPushButton" name="pushButtonDoi">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(215, 226, 255);</string>
     </property>
     <property name="text">
      <string>Đổi</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTaoMoi">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(215, 226, 255);</string>
     </property>
     <property name="text">
      <string>Tạo mới</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonThoat">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(215, 226, 255);</string>
     </property>
     <property name="text">
      <string>Thoát</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
